tiatoolbox/tools/stainextract.py

```diff
@@ -126,23 +124,25 @@ class RuifrokExtractor:
     [https://github.com/Peter554/StainTools] written by Peter Byfield.
 
     Examples:
-        >>> from tiatoolbox.tools.stainextract import RuifrokExtractor()
+        >>> from tiatoolbox.tools.stainextract import RuifrokExtractor
         >>> from tiatoolbox.utils.misc import imread
         >>> extractor = RuifrokExtractor()
         >>> img = imread('path/to/image')
         >>> stain_matrix = extractor.get_stain_matrix(img)
 
     """
 
-    @staticmethod
-    def get_stain_matrix(_):
+    def __init__(self):
+        self.__stain_matrix = np.array([[0.65, 0.70, 0.29], [0.07, 0.99, 0.11]])
+
+    def get_stain_matrix(self, _):
         """Get the pre-defined stain matrix.
 
         Returns:
             ndarray: pre-defined  stain matrix.
 
         """
-        return np.array([[0.65, 0.70, 0.29], [0.07, 0.99, 0.11]])
+        return self.__stain_matrix
```

Comment:
Could possibly return a copy here to avoid the user accidentally modifying the internal stain matrix.

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/268/commits/5148fb900886bc79d5b6b4e9f771679f81a6981e